app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   include Pundit
+  include BrandedExperience
```

Comment:
I usually see concerns used for shared behavior. In this case, it is shared in the sense that it is used in all controllers, but really we are only including it in one (`ApplicationController`), which all other controllers inherit from.

The upside of a concern is that it puts all branded-related stuff in one place. The downside is that it adds indirection -- someone trying to figure out why and how this session metadata is being added will come into the `ApplicatonController` and have to dig around to figure out that it comes from this module that is included and then visit that module. 

If it were me, I would just add those methods to `ApplicatonController` and eliminate `BrandedExperience`. It makes it a longer class, but I think in general the practice of abstracting shared behavior into concerns makes most sense when you want shared behavior, which is not what we have here.